NEWS

```diff
@@ -1,3 +1,7 @@
+* Add access to the new `direction` argument of `scales::brewer_pal` and
```

Comment:
Oh, OK. I thought this is what I always did so I misunderstood your comment regarding this. I'll correct it.

BTW, do you prefer 1-commit pull requests (in which case I overwrite the changes each time and force push, so I guess line comments like this disappear) or many small commits are OK (in which case the "history" of the PR itself is kept but it gets messier when merged).